internal/resources/google/redis_cluster.go

```diff
@@ -9,15 +9,29 @@ import (
 	"github.com/shopspring/decimal"
 )
 
+var redisNodeTypeNames = map[string]string{
+	"REDIS_SHARED_CORE_NANO": "Shared Core Nano",
+	"REDIS_STANDARD_SMALL":   "Standard Small",
+	"REDIS_HIGHMEM_MEDIUM":   "Highmem Medium",
+	"REDIS_HIGHMEM_XLARGE":   "Highmem XLarge",
+}
+
+var redisNodeTypeDescSuffixes = map[string]string{
+	"REDIS_SHARED_CORE_NANO": "Shared Core Nano",
+	"REDIS_STANDARD_SMALL":   "Standard Small",
+	"REDIS_HIGHMEM_MEDIUM":   "Default",
```

Comment:
This fixes an issue with this node type. In the pricing API it comes through as `Redis Cluster Node Default Iowa`, so this fixes that lookup. We still want the name of the cost component to have a clearer name like 'Cluster node (highmem medium)' so I've separated out the name lookups to a separate map.